iris_grib/_save_rules.py

```diff
@@ -611,8 +611,9 @@ def grid_definition_section(cube, grib):
         # Lambert Conformal coordinate system (template 3.30).
         grid_definition_template_30(cube, grib)
     else:
-        raise ValueError('Grib saving is not supported for coordinate system: '
-                         '{}'.format(cs))
+        name = cs.grid_mapping_name.replace('_', ' ').title()
```

Comment:
I don't think the `.title()` should be there : `cs.grid_mapping_name` is just a string.
Presumably you tested this, hence spotting the problem?
Maybe it even needs an actual test.

Reply:
@pp-mo I'm just making the `grid_mapping_name` more human-readable for the sake of the error message, that is:

`lambert_conformal` --> `Lambert Conformal`

Reply:
Sure, but is the method really general?  It depends on the content of the strings.
I think staying closer to the code (the name of the thing) might be actually clearer here.

Reply:
Oh, 'title' is a standard string method  :mortar_board: 
Ok then, I just didn't get that this would actually work !!